src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,497 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['servicebus'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus namespace, queue, topic, subscription, rule and geo disaster recovery configuration - alias
+
+    """
+
+helps['servicebus namespace'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus namespace and authorization-rule
+
+    """
+
+helps['servicebus queue'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus queue and authorization-rule
+
+    """
+
+helps['servicebus topic'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus topic and authorization-rule
+
+    """
+
+helps['servicebus subscription'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus Subscription
+
+    """
+
+helps['servicebus rule'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus rule
+
+    """
+
+helps['servicebus georecovery-alias'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus Geo Disaster Recovery Configuration - Alias
+
+    """
+
+helps['servicebus namespace check-name-availability'] = """
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: Create a new topic.
+          text: az servicebus namespace check_name_availability --name mynamespace
+
+    """
+
+helps['servicebus namespace create'] = """
+    type: command
+    short-summary: Creates the ServiceBus Namespace
+    examples:
+        - name: Create a new namespace.
+          text: helps['az servicebus namespace create --resource-group myresourcegroup --name mynamespace --location westus
+           --tags ['tag1: value1', 'tag2: value2'] --sku-name Standard --sku-tier Standard']
```

Comment:
Bad help text.
Remove the 'helps' and make sure it's a command that can actually be executed.

Reply:
Same below...

Reply:
My Bad, i had observed it but missed to change it. its now resolved